src/game_api/state.cpp

```diff
@@ -238,22 +239,36 @@ State& State::get()
         hook_godmode_functions();
         get_is_init() = true;
         strings_init();
+        init_state_update_hook();
     }
     return STATE;
 }
 
+StateMemory* g_state_ptr{nullptr};
+
 StateMemory* State::ptr() const
 {
+    if (g_state_ptr)
+        return g_state_ptr;
+
     OnHeapPointer<StateMemory> p(read_u64(location));
     return p.decode();
 }
 
 StateMemory* State::ptr_local() const
 {
+    if (g_state_ptr)

```

Comment:
Why? This function is getting another state pointer by design, so you can't take the one that you got from `State::ptr()`